.github/workflows/core-indexer-api.yml

```
@@ -0,0 +1,55 @@
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=match,pattern=core-indexer-api-v(.*),group=1,prefix=v
+            type=ref,event=branch
+
+      - name: Build and push
+        uses: docker/build-push-action@v4
```

actionlint (1.7.7)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)